images/base/Dockerfile

```diff
@@ -91,7 +95,9 @@ RUN echo "Ensuring scripts are executable ..." \
     && tar -C /opt/cni/bin -xzf /tmp/cni.tgz \
     && rm -rf /tmp/cni.tgz \
  && echo "Ensuring /etc/kubernetes/manifests" \
-    && mkdir -p /etc/kubernetes/manifests
+    && mkdir -p /etc/kubernetes/manifests \
+ && echo "Adjusting systemd-tmpfiles timer" \
+    && sed -i /usr/lib/systemd/system/systemd-tmpfiles-clean.timer -e 's#OnBootSec=.*#OnBootSec=1min#'
```

Comment:
the default is 15m after boot and then every 24 hours.
for now just moving it to 1m after boot which imo makes more sense for kind nodes which generally live on a compressed times cale, and leaving the 24 hours alone.

the real fix is hopefully in the /var/tmp change :upside_down_face: